src/main/java/uk/ac/ebi/eva/evaseqcol/controller/seqcol/SeqColController.java

```diff
@@ -28,9 +31,25 @@ public SeqColController(SeqColService seqColService) {
         this.seqColService = seqColService;
     }
 
+    @Operation(summary = "Retrieve seqCol object by digest",
+    description = "Given a seqCol's level 0 digest, this endpoint will try to fetch the corresponding seqCol object from" +
+            " the database and return it in the specified level. The default level value is 1.")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "SeqCol was returned successfully"),
+            @ApiResponse(responseCode = "404", description = "Could not find a seqCol object with the given digest"),
+            @ApiResponse(responseCode = "400", description = "Not valid level value"),
+            @ApiResponse(responseCode = "500", description = "Server Error")
+    })
     @GetMapping(value = "/{digest}")
     public ResponseEntity<?> getSeqColByDigestAndLevel(
-            @PathVariable String digest, @RequestParam(required = false) String level) {
+            @Parameter(name = "digest",
+            description = "SeqCol's level 0 digest",
+            example = "3mTg0tAA3PS-R1TzelLVWJ2ilUzoWfVq",
+            required = true) @PathVariable String digest,
+            @Parameter(name = "level",
+            description = "The desired output's level",
+            example = "1 or 2",
```

Comment:
I would do it this way, as I think the UI uses the example values to auto-fill sample requests.
```suggestion
            description = "The desired output's level (1 or 2)",
            example = "1",
```